api/api/src/test/scala/uk/ac/wellcome/platform/api/works/ApiWorksTestBase.scala

```diff
@@ -89,6 +89,16 @@ trait ApiWorksTestBase
       "description": "$description"
     }"""
 
+  def goneRequest(apiPrefix: String, description: String) =
```

Comment:
If you’re creating this, might be worth modifying `gone()` (L130–8) to use this.

Reply:
Updated `gone` to be `deleted` to be more specific.